File: api/methods/motd.ts
import Rand from 'rand-seed'
import Monster from '../interfaces/monster'
import { monsterList } from '../data/monsterList'

export default function getMonsterOfTheDay(): Monster | null {
  let seed = new Date().toDateString()
  if (!monsterList) { return null }
  return monsterList[ monsterList.length * new Rand(seed).next() << 0 ]
}

File: api/methods/motd.py
import random

from datetime import datetime

from interfaces.Monster import Monster

def monsterOfTheDay(monster_list: list[Monster]) -> Monster:
    monsterNames = [monster['name'] for monster in monster_list]
    seed = datetime.today().strftime("%d/%m/%Y")
    random.seed(seed)
    random.shuffle(monsterNames)
    monster_of_the_day = random.choice(monsterNames)
    return [monster for monster in monster_list if monster['name'] == monster_of_the_day][0]
